ugvc/__main__.py

```diff
@@ -55,7 +55,10 @@
 from ugvc.pipelines.sec import assess_sec_concordance, correct_systematic_errors, sec_training, sec_validation
 from ugvc.pipelines.vcfbed import annotate_contig, calibrate_bridging_snvs, intersect_bed_regions
 from ugvc.scripts import sorter_to_h5
-from ugvc.somatic_cnv import bicseq2_post_processing
```

Comment:
Can you merge cnv and somatic_cnv please? There is no real reason for the split, right?

Reply:
moved to ugbio-cnv